src/components/Checkbox/Checkbox.scss

```diff
@@ -5,199 +5,199 @@
 //
 // Office UI Fabric
 // --------------------------------------------------
-// Choice fields (radio buttons and checkboxes) styles
-
-// Unselected, radio button (default)
-.ms-ChoiceField {
-  @include ms-font-m;
-  @include ms-u-normalize;
+// Checkbox styles
+
+$ms-checkbox-field-size: 20px;
+$ms-checkbox-transition-duration: 200ms;
```

Comment:
Not in the scope of this PR to change now, but boy, it would be nice if these animation values could be reconciled against what's in the animation library (or rather, have the animation library updated so we don't have one-off values like throughout the components).